src/content/docs/browser/browser-integrations/mexn-integration.mdx

```diff
@@ -4,18 +4,19 @@ tags:
     - New Relic integrations
     - MEAN stack integration
     - MERN stack integration
+    - MEVN stack integration
 metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your MExN stack.
 ---
 
 
-import infrastructureMeanMernIntegrationDashboard from 'images/infrastructure_screenshot-full_mean-mern-integration-dashboard.webp'
+import infrastructureMeanMernMevnIntegrationDashboard from 'images/infrastructure_screenshot-full_mean-mern-integration-dashboard.webp'
```

Comment:
don't know if it matters but the name of image was changed but the actual image wasn't changed. but maybe that's as desired